### ISSUE TITLE
input focus

### ISSUE THREAD

**Issue by yangmeishu** [hansalemaos/input_timeout#3]
windows  10 system 
When you enter a character and haven't pressed Enter, but the window is switched to another program, the system's input focus will remain in the input terminal. This will cause other programs to react to the phenomenon that one character is entered and one character is deleted. The input focus is forcibly occupied.

**Comment by hansalemaos** [comment]
Yes, that is true. I haven't found any other way to solve this problem.
